.github/workflows/scala3doc.yaml

```diff
@@ -0,0 +1,58 @@
+name: CI for Scala3doc
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: lampepfl/dotty:2020-04-24
+    
+    steps:
+
+      - uses: actions/checkout@v2
+      - run: git fetch --prune --unshallow --tags
+
+      - name: Cache Coursier
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/coursier
+          key: sbt-coursier-cache
+      - name: Cache SBT
+        uses: actions/cache@v1
+        with:
+          path: ~/.sbt
+          key: sbt-${{ hashFiles('**/build.sbt') }}
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Compile and test
+        run: ./project/scripts/sbt scala3doc/test
+
+      - name: Locally publish self
+        run: ./project/scripts/sbt scala3doc/publishLocal
+
+      - name: Generate test documentation
+        run: ./project/scripts/sbt scala3doc/generateSelfDocumentation
+
+      - name: Generate Scala 3 documentation
+        run: ./project/scripts/sbt scala3doc/generateScala3Documentation
+
+      - name: Generate documentation for example project using dotty-sbt
+        run: |
+          ./project/scripts/sbt ";sbt-dotty/publishLocal;scala3/publishLocal;publishLocal"
+          VERSION=$(./project/scripts/sbt "show sbt-dotty/version" | tail -1 | sed 's/\[info\] //')
+          echo Using sbt-dotty version: $VERSION
+          echo 'addSbtPlugin("ch.epfl.lamp" % "sbt-dotty" % "'$VERSION'")' > scala3doc/example-project/project/dotty.sbt
+
+          SCALA_VERSION=$(./project/scripts/sbt "show scala3-compiler-bootstrapped/version" | tail -1 | sed 's/\[info\] //')
+          echo Using scala version: $SCALA_VERSION
+          echo 'scalaVersion := "'$SCALA_VERSION'"' > scala3doc/example-project/dotty.sbt
+
+          cd scala3doc/example-project
+          sbt doc
```

Comment:
Isn't this basically equivalent to running the sbt scripted test [`sbt-dotty/scala3doc`](https://github.com/romanowski/dotty/tree/scala3doc/migration/sbt-dotty/sbt-test/sbt-dotty/scala3doc)?
```scala
sbt "sbt-dotty/scripted sbt-dotty/scala3doc"
```

Reply:
Yes, the only difference is that the project is bigger but I can move it there.